src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -0,0 +1,255 @@
+          identity.id.toLowerCase().includes(q),
+      )
+    ) {
+      return false;
+    }
+
+    if (
+      filters.authMethod.length > 0 &&
+      !filters.authMethod.includes(identity.authentication_method)
+    ) {
+      return false;
+    }
+
+    return true;
+  });
+
+  const rows = filteredIdentities.map((identity) => {
+    const isLoggedInIdentity = settings?.auth_user_name === identity.id;
+    return {
+      name: identity.name,
+      className: "u-row",
+      columns: [
+        {
+          content: (
+            <Link
+              className="u-truncate name-link"
+              title={`name: ${identity.name}`}
+              to={`/ui/permissions/identity/${identity.authentication_method}/${encodeIdentityNameForUrl(identity.name)}`}
+            >
+              {identity.name}
+              {isLoggedInIdentity && (
+                <Chip value="You" className="u-no-margin--bottom" />
+              )}
+            </Link>
+          ),
+          role: "cell",
+          "aria-label": "Name",
+        },
+        {
+          content: identity.id,
+          role: "cell",
+          "aria-label": "ID",
+          className: "u-truncate",
+          title: `id: ${identity.id}`,
+        },
+        {
+          content: identity.authentication_method.toUpperCase(),
+          role: "cell",
+          "aria-label": "Auth method",
+        },
+        {
+          content: identity.type,
+          role: "cell",
+          "aria-label": "Type",
+          className: "u-truncate",
+          title: `type: ${identity.type}`,
+        },
+        {
+          content: identity.groups?.length || 0,
+          role: "cell",
+          className: "u-align--right",
+          "aria-label": "Groups for this identity",
+        },
+      ],
+      sortData: {
+        id: identity.id,
+        name: identity.name,
+        authentication_method: identity.authentication_method,
+        type: identity.type,
+        groups: identity.groups?.length || 0,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({ rows });
+
+  const selectedIdentities = identities.filter((identity) =>
+    selectedIdentityNames.includes(identity.name),
+  );
+
+  // NOTE: tls user group membership cannot be modified, this will be supported in the future
+  const unSelectableIdentityNames = new Set<string>();
+  const nonTlsUsers = identities.filter((identity) => {
+    const isTlsIdentity = identity.authentication_method === "tls";
+    if (isTlsIdentity) {
+      unSelectableIdentityNames.add(identity.name);
+    }
+    return !isTlsIdentity;
+  });
+
+  if (isLoading) {
+    return <Loader text="Loading identities" />;
+  }
+
+  return (
+    <>
+      <CustomLayout
+        contentClassName="u-no-padding--bottom"
+        header={
+          <PageHeader>
+            <PageHeader.Left>
+              <PageHeader.Title>
+                <HelpLink
+                  href={`${docBaseLink}/reference/manpages/lxc/auth`}
```

Comment:
We might have a better link for identities? @ru-fu should know.

Reply:
I'm missing the context a bit, but probably this page: https://documentation.ubuntu.com/lxd/en/latest/explanation/authorization/